### PR TITLE
!lmgtfy or !google search term

### PR DESCRIPTION
**Pull request by aatrey882**
Slack will return either lmgtf(Let me google for you) or google search result link...

### CODE REVIEW

**Comment by earlduque** [comment]
Hi @aatrey882 at first glance,

your regex activation is `(SN)||(HCLTECH)||(SERVICENOW)` which doesn't match the intent of the parser, please take a look and/or make sure to test in your own instance/bot to verify everything works as intended. Once fixed you can reopen this PR or submit a new PR.

Note that I only reviewed up to when I found the first error.

File: Parsers/!lmgtfy or !google search term.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+} else if (current.text.indexOf('!google') > -1) {
+    where = current.text.indexOf('!google') + 8;
+    which = 'google';
+}
+
+var term = current.text.substr(where).trim();
+
+if (term == '') {
+    var send_confusion = new x_snc_slackerbot.Slacker().send_chat(current.channel, ':upside_down_face: !docs *something*', false, '', current.thread_ts);
```
Comment (chelming) [inline Parsers/!lmgtfy or !google search term.js:24]: I don't think you should be referencing `!docs` here

File: Parsers/!lmgtfy or !google search term.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/*
+activation_example:!lmgtfy or !google search term 
+regex:(SN)||(HCLTECH)||(SERVICENOW)
```
Comment (chelming) [inline Parsers/!lmgtfy or !google search term.js:3]: your regex should match your activation example

File: Parsers/!lmgtfy or !google search term.js (hunk 1 above)
Comment (aatrey882) [reply Parsers/!lmgtfy or !google search term.js:24]: Yes it is referencing if term is empty

Comment (chelming) [reply Parsers/!lmgtfy or !google search term.js:24]: if the term was empty you wouldn't want the user to type `!docs ${their query}`. You want them to type `!lmgtfy` or `!google` so your message should reflect that, probably with something like:
```javascript
':upside_down_face: !' + where + ' *something*'
```

**Comment by aatrey882** [comment]
Resolved

**Pull request closed without merging**